volatility3/framework/plugins/windows/svcscan.py

```diff
@@ -232,28 +232,40 @@ def _get_service_binary_map(
             for service_key in services
         }
 
-    def _generator(self):
-        kernel = self.context.modules[self.config["kernel"]]
+    def enumerate_vista_or_later_header(
```

Comment:
Again, this isn't following the standard `classmethod` and parameterized context as literally every other core plugin.  Please adapt it to fit in with the way all the other plugins play together...  5;)